### PR TITLE
Described DBaaS feature in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by gen1us2k**
PMM-0

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]

**Review by oksana-grishchenko — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,6 +47,31 @@ This project is built from several repositories:
...
+    ENABLE_DBAAS=1 NETWORK=minikube make env-up # Run PMM with DBaaS feature enabled
+```
+
+[Read the documentation](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/dbaas.html) how to run DBaaS on GKE or EKS
+
+##### Troubleshooting
+
+1. You can face with pod failing with `Init:CrashLoopBackOff` issue. Once you get logs by running `kubectl logs pxc-cluster-pxc-0 -c pxc-init` you get the error `install: cannot create regular file '/var/lib/mysql/pxc-entrypoint.sh': Permission denied`. You can fix it using [this solution](https://github.com/kubernetes/minikube/issues/12360#issuecomment-1123794143). Also, check [this issue](https://jira.percona.com/browse/K8SPXC-879)
+1. Multinode PXC Cluster can't be created on ARM CPUs. You can have single node installation.
+1. Operators are not supported. This issue can happen in two different scenarios. You can have PMM version higher then current release, our you installed higher version of operators. You can check compatibility using https://check.percona.com/versions/v1/pmm-server/PMM-version
```
Comment (BupycHuk) [inline CONTRIBUTING.md:72]:
```suggestion
1. Operators are not supported. It means that the PMM version <-> operator version pair does not exist in the Version service. This issue can happen in two different scenarios. You can have a PMM version higher than the current release, or you installed a higher version of operators. You can check compatibility using https://check.percona.com/versions/v1/pmm-server/PMM-version
```

Comment (BupycHuk) [inline CONTRIBUTING.md:72]: all items have the number 1.

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (gen1us2k) [reply CONTRIBUTING.md]: Yep. But it renders correctly by github.

Comment (BupycHuk) [reply CONTRIBUTING.md]: But not locally :)

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**